generate/metricbeat/metricset/{{cookiecutter.beat}}/Makefile

```diff
@@ -11,10 +11,17 @@ PREFIX?=.
 
 # Initial beat setup
 .PHONY: setup
-setup:
+setup: copy-vendor
 	make create-metricset
 	make collect
 
+# Copy beats into vendor directory
+.PHONY: copy-vendor
+copy-vendor:
+	mkdir -p vendor/github.com/elastic/
+	cp -R ${GOPATH}/src/github.com/elastic/beats vendor/github.com/elastic/
+	rm -rf vendor/github.com/elastic/beats/.git
```

Comment:
Consider also calling the `scripts/clean-vendor.sh` script to reduce the number of files.